spec/models/item_spec.rb

```diff
@@ -17,6 +17,36 @@
     it {should have_many(:orders).through(:item_orders)}
   end
 
+  describe "class methods" do
+    before(:each) do
+
+      @meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      @brian = Merchant.create(name: "Brian's Dog Shop", address: '125 Doggo St.', city: 'Denver', state: 'CO', zip: 80210)
+      @tire = @meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
+      @pull_toy = @brian.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
+      @dog_bone = @brian.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
+      @bone = @brian.items.create(name: "A Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
+      @toy = @brian.items.create(name: "A Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
+      @order = Order.create!(name: "name", address: "address", city: "city", state: "state", zip: 23455)
+      @order2 = Order.create!(name: "name", address: "address", city: "city", state: "state", zip: 23455)
+      ItemOrder.create!(order_id: @order.id, price: 1.0, item_id: @dog_bone.id, quantity: 5)
+      ItemOrder.create!(order_id: @order.id, price: 1.0, item_id: @pull_toy.id, quantity: 1)
+      ItemOrder.create!(order_id: @order.id, price: 1.0, item_id: @tire.id, quantity: 4)
+      ItemOrder.create!(order_id: @order.id, price: 1.0, item_id: @toy.id, quantity: 3)
+      ItemOrder.create!(order_id: @order.id, price: 1.0, item_id: @bone.id, quantity: 2)
+      ItemOrder.create!(order_id: @order2.id, price: 1.0, item_id: @dog_bone.id, quantity: 3)
+      ItemOrder.create!(order_id: @order2.id, price: 1.0, item_id: @pull_toy.id, quantity: 4)
+    end
+
+    it "can find and order top 5 items" do
+      expect(Item.top_5).to eq({ @dog_bone.name => 8, @pull_toy.name => 5, @tire.name => 4, @toy.name => 3, @bone.name => 2})
+    end
+
+    it "can find and order worst 5 items" do
+      expect(Item.worst_5).to eq({ @bone.name => 2, @toy.name => 3, @tire.name => 4, @pull_toy.name => 5, @dog_bone.name => 8 })
+    end
```

Comment:
Oops, forgot to add these the other day!

Reply:
Good catch.